### PR TITLE
OWASP top ten 2021 explained with non-technical examples, Part 1

### PR DESCRIPTION
**Pull request by petteripoyhtari**
How could we better make the most typical risks of web applications aware of all project personnel, in order to improve the quality of the entire web infrastructure. A blog post directed to non-technical people about OWASP top 10 with physical world analogies. First one to go 12.1. and second one 26.1.

### CODE REVIEW

**Review by jarnovayrynen — APPROVED** [review]

File: _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## Current cyber weather
+
+Not surprisingly, cyber security should be taken more seriously now than ever. Statistics show (e.g. a report by [Google's threat analysis group](https://security.googleblog.com/2023/07/the-ups-and-downs-of-0-days-year-in.html)) that the number of zero-day vulnerabilities is growing exponentially. Also, the use of 0-day in August 2023 exceeds the figure for the entire year of 2022.
+
+![Infra in cyber weather](/img/2024-01-owasp-top-10/0_cyber_weather_2.jpg)
+
+Does this say more about the global cyber situation or the bad code we produce? Who is to blame here? Juhani Eronen, the leading expert at Traficom's Cybersecurity Center [stated](https://www.kyberturvallisuuskeskus.fi/fi/ajankohtaista/miten-ohjelmistokehityksen-turvallisuutta-voidaan-kehittaa-tuore-selvitys-kartoitti), "*According to our observations, poor software quality is one of the biggest security problems.*"
```
Comment (jarnovayrynen) [inline _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md:27]: Maybe very briefly state what Traficom is, for non Finnish audience, when mentioning it the first time?

File: _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+![Different roles](/img/2024-01-owasp-top-10/0_motivation.jpg)
+
+## OWASP to the rescue!
+
+There are many good checklists and tables for improving project and product safety (eg [ASVS](https://owasp.org/www-project-application-security-verification-standard/) and [SAMM](https://owaspsamm.org/)). One of these lists is the well-known list from our dear good friend [Open Worldwide Application Security Project top 10](https://owasp.org/www-project-top-ten/).
+
+OWASP is a non-profit foundation dedicated to improving software security. Every two years, they provide a list of the most critical security risks for web applications. The 2023 version is in the release candidate stage and this blog post focuses on the 2021 list risks. You can also increase your motivation by reading Riika's awesome blog post [Why a developer should know OWASP](https://dev.solita.fi/2020/12/15/why-developer-should-know-owasp.html).
```
Comment (jarnovayrynen) [inline _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md:49]: Riikka's

Comment (petteripoyhtari) [reply _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md]: fixed

File: _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md (hunk 1 above)
Comment (petteripoyhtari) [reply _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md:27]: Good point. I'll do that.

Comment (petteripoyhtari) [reply _posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md]: Oisko nyt hyvä? Lisäsin tiivistetyn maininnan + linkit for furhter information.

**Review by jarnovayrynen — APPROVED** [review]

**Pull request merged**